shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -39,20 +39,16 @@
  * <p>
  * <strong>Dart entrypoint, initial route, and app bundle path</strong>
  * <p>
- * The Dart entrypoint executed within this {@code Activity} is "main()" by default. The entrypoint
- * may be specified explicitly by passing the name of the entrypoint method as a {@code String} in
- * {@link #EXTRA_DART_ENTRYPOINT}, e.g., "myEntrypoint".
+ * The Dart entrypoint executed within this {@code Activity} is "main()" by default. To change the
+ * entrypoint that a {@code FlutterActivity} executes, subclass {@code FlutterActivity} and
+ * override {@link #getDartEntrypointFunctionName()}.
```

Comment:
or start your own engine?

Reply:
I added a section to the cached engine part that refers to this:
https://github.com/flutter/engine/pull/11239/files#diff-d523f6ab2ed9834f6b3232b67755b7d5R76